### PR TITLE
Delete Hydrolysis elements and fix Polymer 2 compatibility.

### PR DESCRIPTION
**Pull request by aomarks**
- Delete Hydrolysis elements, since the docs site (soon) no longer uses them.
- Drop all the `-2` suffixes for the Analyzer elements.
- Rename `prefix` to `fragmentPrefix` because `prefix` is reserved. This fixes `<iron-doc-viewer>` under Polymer 2, so I think it's *actually* hybrid after this.
- Fix a test breakage.

Note this can't be merged until https://github.com/Polymer/docs/pull/2184 is in, since otherwise the docs site will break.

### CODE REVIEW

File: demo/analyzer.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
     <!-- <script src="../../webcomponentsjs/webcomponents-lite.js"></script> -->
     <link rel="import" href="../../polymer/polymer.html">
-    <!--<link rel="import" href="../iron-doc-viewer-2.html">-->
+    <!--<link rel="import" href="../iron-doc-viewer.html">-->
```
Comment (samuelli) [inline demo/analyzer.html]: I know its not from your change, but can we clean up these commented out lines?

File: index.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
     <!-- <script src="../../webcomponentsjs/webcomponents-lite.js"></script> -->
```
Comment (samuelli) [inline index.html]: Here too

File: iron-doc-property.html [hunk 3]
```diff
@@ -76,6 +80,11 @@
             observer: '_descriptorChanged',
           },
 
+          function: {
+            type: Boolean,
+            reflectToAttrobute: true,
```
Comment (samuelli) [inline iron-doc-property.html]: `reflectToAttribute`

**Review by samuelli — APPROVED** [review]

File: iron-doc-property.html (hunk 3 above)
Comment (aomarks) [reply iron-doc-property.html]: Done.

File: demo/analyzer.html (hunk 1 above)
Comment (aomarks) [reply demo/analyzer.html]: Done.

File: index.html (hunk 2 above)
Comment (aomarks) [reply index.html]: Done.

**Review by aomarks — COMMENTED** [review]
Thanks!

**Pull request merged**